src/test/java/com/google/firebase/projectmanagement/AndroidAppTest.java

```diff
@@ -189,6 +189,23 @@ public void testDeleteShaCertificateAsync() throws Exception {
     Mockito.verify(androidAppService).deleteShaCertificateAsync(CERTIFICATE_NAME);
   }
 
+  @Test
+  public void testAndroidAppMetadata() {
+    assertEquals(APP_NAME, ANDROID_APP_METADATA.getName());
+    assertEquals(APP_ID, ANDROID_APP_METADATA.getAppId());
+    assertEquals(APP_DISPLAY_NAME, ANDROID_APP_METADATA.getDisplayName());
+    assertEquals(PROJECT_ID, ANDROID_APP_METADATA.getProjectId());
+    assertEquals(APP_PACKAGE_NAME, ANDROID_APP_METADATA.getPackageName());
+  }
+
+  @Test
+  public void testAndroidAppMetadataEquality() {
+    AndroidAppMetadata other =
+        new AndroidAppMetadata(APP_NAME, APP_ID, APP_DISPLAY_NAME, PROJECT_ID, APP_PACKAGE_NAME);
```

Comment:
nit: Let's stay consistent with the rest of the test and insert a blank line here between the Arrange section and Act + Assert combo section.

Reply:
Done